physionet-django/console/templates/console/credential_processing.html

```diff
@@ -22,10 +22,10 @@
         <a class="nav-link" id="reference-tab" data-toggle="tab" href="#reference" role="tab" aria-controls="reference" aria-selected="false">Reference Check {{ reference_applications|task_count_badge|safe }}</a>
       </li>
       <li class="nav-item">
-        <a class="nav-link" id="response-tab" data-toggle="tab" href="#response" role="tab" aria-controls="response" aria-selected="false">Reference Response {{ response_applications|task_count_badge|safe }}</a>
+        <a class="nav-link" id="response-tab" data-toggle="tab" href="#response" role="tab" aria-controls="response" aria-selected="false">Awaiting Reference Response {{ response_applications|task_count_badge|safe }}</a>
```

Comment:
Does this cause any issues with page width? If not, no problem, otherwise perhaps "Awaiting response" would be okay.

Reply:
Yes, no problem with page width.